apps/etterlatte-behandling/src/test/kotlin/BehandlingIntegrationTest.kt

```diff
@@ -189,7 +189,7 @@ abstract class BehandlingIntegrationTest {
         HttpClient(MockEngine) {
             engine {
                 addHandler { request ->
-                    if (request.url.fullPath.matches(Regex("api/grunnlag/[0-9]{11}"))) {
+                    if (request.url.fullPath.matches(Regex("api/grunnlag/sak/[0-9]+"))) {
```

Comment:
Vi støtter vel ikke sakider som starter på `0`?

Reply:
Lite trolig, men dette er bare for å fange alle dummy-verdier i testen. Så ikke så nøye 🤷‍♂️